Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -217,10 +286,20 @@ def customLogLevel(custom: acts.logging.Level = acts.logging.INFO):
                     rMax=seedfinderConfigArg.r[1],
                     deltaRMin=seedfinderConfigArg.deltaR[0],
                     deltaRMax=seedfinderConfigArg.deltaR[1],
-                    deltaRMinTopSP=seedfinderConfigArg.deltaR[0],
-                    deltaRMinBottomSP=seedfinderConfigArg.deltaR[0],
-                    deltaRMaxTopSP=seedfinderConfigArg.deltaR[1],
-                    deltaRMaxBottomSP=seedfinderConfigArg.deltaR[1],
+                    deltaRMinTopSP=seedfinderConfigArg.deltaR[0]
+                    if seedfinderConfigArg.deltaRTopSP[0] == None
+                    else seedfinderConfigArg.deltaRTopSP[0],
```

Comment:
```suggestion
                    deltaRMinTopSP=(
                        seedfinderConfigArg.deltaR[0]
                        if seedfinderConfigArg.deltaRTopSP[0] is None
                        else seedfinderConfigArg.deltaRTopSP[0]
                    ),
```
The usual recommendation is to use `is None` instead of `== None`.
I think it's a bit clearer to put the whole expression in `(...)`, but it's fine if you prefer without.

Reply:
done!